extensions/amp-bind/0.1/test/integration/test-bind-impl.js

```diff
@@ -983,13 +983,8 @@ describe
               mystate: {mykey: 'myval'},
             });
 
-            // Integration tests may not use chai-as-promised.
-            try {
-              await bind.getStateAsync('mystate.mykey');
-              expect.fail();
-            } catch (err) {
-              expect(err).match(/#mystate/);
-            }
+            const state = bind.getStateAsync('mystate.mykey');
+            expect(state).to.eventually.rejectedWith(/#mystate/);
```

Comment:
Doesn't this need to be returned, or a `done()` callback called, for mocha to handle a promise rejection?

Reply:
Ah, I didn't realize that. Fixed!